src/checkout/checkout-service.ts

```diff
@@ -553,9 +557,12 @@ export default class CheckoutService {
     /**
      * Continues to check out as a guest.
      *
-     * The customer is required to provide their email address in order to
-     * continue. Once they provide their email address, it will be stored as a
-     * part of their billing address.
+     * If your checkout settings allow it, your customers could continue the checkout as guests (without signing in).
```

Comment:
I looked through the document, seems like we are using `Checkout Settings` by capitalize the first letter for each word